src/v1/theme/style-overrides/component-overrides/button-overrides.ts

```diff
@@ -16,7 +16,10 @@ const generateButtonOverrides = (uiTheme: UITheme) =>
           border: `1px solid ${theme.palette.primary.main}`,
           backgroundColor: theme.palette.secondary.main,
           ":hover": {
-            backgroundColor: `${THEME_COLORS[uiTheme][100]}`,
+            backgroundColor:
+              uiTheme === "light" || "dark"
```

Comment:
Bit confused about what this is doing - should it be
```suggestion
              uiTheme === "light" || uiTheme === "dark"
```
?